snapshot/example/fastlane/Snapfile

```diff
@@ -1,17 +1,24 @@
 # Uncomment the lines below you want to change by removing the # in the beginning
 
 # A list of devices you want to take the screenshots from
-devices([
-   "iPhone SE (2nd generation)", 
-   "iPhone 12 Pro Max"
-])
-
- languages([
-  "en-US",
-  "de-DE",
-  "it-IT",
-#   ["pt", "pt_BR"] # Portuguese with Brazilian locale
- ])
+devices(
+  [
+    "iPhone 6s",
+    "iPad Air"
```

Comment:
**This was force change** to make Xcode8 spec happy `test_command_generator_xcode_8_spec.rb` 😇
it seems, our snapshot unit tests require some attention, I will have a look in the coming week 💪